qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local.properties

```diff
@@ -7,4 +7,4 @@ login.auto=false
 headless=true
 db.host=openmrs
 webdriver.gecko.driver=firefoxdriver/linux/geckodriver
-includes.csrftoken=true
+includes.csrftoken=true
```

Comment:
Any reason for this change?

Reply:
No change here checkout this line https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local.properties#L10